tests/elis_api_client/test_documents.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import pytest
+
+from rossum_api.api_client import Resource
+
+
+@pytest.mark.asyncio
+class TestQueues:
```

Comment:
```suggestion
class TestDocumentsAsync:
```

Reply:
Using `TestDocuments` to make it consistent with all other tests